### DISCUSSION TITLE
FTS method

### DISCUSSION THREAD

**Discussion by Samlesh** [MICCoM/SSAGES-public#33] [Q&A]
Does the SSAGES software give an option to use metric tensor M value anything apart from unity in FTS? What are your views as to how different the results would be?

**Comment by p-zubieta** [comment]
Moving this to discussions. We can add an issue later if deemed necessary.

**Comment by p-zubieta** [comment]
Following up with your question. Right now there's no way to pass an option that would compute the metric tensor and use it. Using the euclidean distances as we do should give you close enough paths in many cases. That said, we would welcome additions to the code and are interested in adding it if it would be more useful to the community.

**Reply by Samlesh** [reply]
Thanks, Pablo for your answer. I was looking to use FTS with a collective variable such as pairwise distance and angle between two sets of groups in the protein. Hopefully, I get close reactive pathways using the FTS method, as indicated in your response.

**Reply by p-zubieta** [reply]
No problem. Feel free to follow up the discussion if you have any issues with it.

**Reply by Samlesh** [reply]
Dear Pablo,
I was executing the examples section in the FTS folder of examples on Gromacs 5.1.3. I could get the desired results of examples having ADP in vacuum for both a single processor and two processors per walker. However, when I tried to run the ADP water environment with the provided adp_H2O.tpr file, Gromacs gave an error of could not settle bad contacts with water. So I did an energy minimization step externally on the provided gro file initially and followed the same procedure in README.md to execute. However, the same bad contact error popped up after running 8 ps. What I think is when the new image position is generated, I am doubtful if the software is having an energy minimization inbuilt to remove bad contacts. Can you please help me out to run this example of ADP in water in Gromacs by sorting this error?
